kiosk/src/Models/Kiosk.ts

```diff
@@ -168,7 +186,30 @@ export class Kiosk {
         this.siteElements.forEach(item => gamespace.appendChild(item));
     }
 
+    getBuiltGame(gameId: string): BuiltSimJSInfo | void {
```

Comment:
```suggestion
    getBuiltGame(gameId: string): BuiltSimJSInfo | undefined {
```

typically void as a return value means "do not try to read the return value" where using undefined signifies "you can try, but it might fail"